master/buildbot/steps/shell.py

```diff
@@ -200,6 +201,8 @@ def _describe(self, done=False):
             words = self.command
             if isinstance(words, (str, unicode)):
                 words = words.split()
+            elif isinstance(words, Interpolate):
+                return str(words)
```

Comment:
Are you sure it is safe to return a string here?

The rest of code is always returning lists of strings.

Reply:
You're right. I did not notice a type for return value.